packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -157,7 +157,7 @@
 						$locale.baseText('nodeSettings.nodeVersion', {
 							interpolate: {
 								node: nodeType?.displayName as string,
-								version: node.typeVersion.toString(),
+								version: (node.typeVersion ?? latestVersion).toString(),
```

Comment:
awesome. thanks for the fix. Would be great to add a test that would catch this issue.